build/8-to-9-migration/optaplanner-quarkus3.yaml

```diff
@@ -14,13 +14,21 @@ recipeList:
       configuration: <skip>true</skip>
   - org.openrewrite.maven.ChangePropertyValue:
       key: version.io.quarkus
-      newValue: 3.0.0.CR1
+      newValue: 3.0.0.Final
```

Comment:
This will prove painful. It will require manual updates, or another piece of CI.

Reply:
It already is painful, believe me. I will try the version expression the `UpgradeDependencyVersion` supports; perhaps it will work for version properties as well.

Reply:
Unfortunately, we have to keep the property, as we use it in multiple places, not only for dependencies.